cli50

```diff
@@ -1,7 +1,8 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
@crossroads1112 is this the right way to ensure Python 3 is used, even if both 2 and 3 are installed?